src/Facade/Common/AsyncFacadeService.cs

```diff
@@ -264,7 +264,8 @@ protected abstract void DeleteOrObsoleteResource(
             T entity,
             IEnumerable<string> privileges = null);
 
-        protected abstract Expression<Func<T, bool>> FilterExpression(TSearchResource searchResource);
+        protected abstract Expression<Func<T, bool>> FilterExpression(TSearchResource searchResource,
```

Comment:
I like the idea of passing privs to FilterBy, but since authorisation is still using the older facade service code we will need to make this change in Linn.Common.Facade too if Oliver wants to copy this example